.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -7,9 +7,11 @@ body:
     attributes:
       value: |
         Thank you for taking time to suggest your idea! Please be aware that this process takes a while (at least weeks). We appreciate your patience!
-        
+
         If you haven't already, [please go search the existing issues for ideas similar to yours](https://github.com/django/new-features/issues?q=is%3Aissue).
-        
+
+        Please use a single issue for a one feature suggestions. If you have related ideas, please create multiple issues which reference each other.
```

Comment:
"Please use a single issue for a one feature suggestions" needs a reword, I think!